src/app/api/companies/route.ts

```diff
@@ -0,0 +1,20 @@
+import { NextResponse, NextRequest } from 'next/server'
+
+export async function GET(req: NextRequest) {
+  const companyId = req.nextUrl.searchParams.get('companyId') as string
+
+  const options = {
+    method: 'GET',
+    headers: {
+      accept: 'application/json',
+      'X-API-KEY': process.env.COPILOT_API_KEY as string,
+    },
+  }
+
+  const res = await fetch(
+    `https://api-beta.copilot.com/v1/companies/${companyId}`,
+    options,
+  )
+  const company = await res.json()
+  return NextResponse.json({ data: company })
+}
```

Comment:
@aatbip Looks like the function is fetching single company details, so api folder structure should reflect that, like `api/companies/[:id]/route.ts`